### PR TITLE
Fixed Issue #483 for a test done for fs.mknod promise

### PR DESCRIPTION
**Pull request by mskuybeda**
used the existing test for fs and adjusted it to promise based version. 
it creates a directory to test a node at path

### CODE REVIEW

File: tests/spec/fs.mknod.spec.js [hunk 1]
```diff
@@ -1,82 +1,94 @@
...
+      if (error) throw error;
+
+      fs.stat('/file', function (error, result) {
         expect(error).not.to.exist;
         expect(result.type).to.equal('FILE');
         done();
       });
     });
   });
+  describe.only('Promise test mknod', function () {
```
Comment (SeanPrashad) [inline tests/spec/fs.mknod.spec.js]: I think it's OK to keep your newly added test inside the existing `describe` block - other students have been putting them there!

File: tests/spec/fs.mknod.spec.js [hunk 2]
```diff
@@ -1,82 +1,94 @@
 var util = require('../lib/test-utils.js');
 var expect = require('chai').expect;
 
-describe('fs.mknod', function() {
+describe('fs.mknod', function () {
```
Comment (SeanPrashad) [inline tests/spec/fs.mknod.spec.js]: Let's consider cleaning up the white spaces in a separate PR! We should aim to keep PR's confined to one specific thing - in this case, it's simply adding a new test as stated in #483. Another issue should be opened with a title of `Remove whitespace in test files` if you'd like!

With that being said, can you please revert the white space changes in a new commit? Please let me know if you need any help with this! 😁

File: tests/spec/fs.mknod.spec.js [hunk 3]
```diff
@@ -1,82 +1,94 @@
...
         expect(result.type).to.equal('FILE');
         done();
       });
     });
   });
+  describe.only('Promise test mknod', function () {
+
+    it('should return an error if part of the parent path does not exist', function () {
+      var fs = util.fs().promises;
+
```
Comment (SeanPrashad) [inline tests/spec/fs.mknod.spec.js]: We can remove this blank line ✌️

Comment (mskuybeda) [reply tests/spec/fs.mknod.spec.js]: solved

**Comment by SeanPrashad** [comment]
@mskuybeda I definitely agree - this calls for [Prettier](https://prettier.io/)

**Comment by humphd** [comment]
@mskuybeda I'm going to land this, but it's not good that you modified all those other lines for style-only changes, and then didn't reset them.  In the future, be very careful about this, and don't let this type of thing get into your PRs.

**Pull request merged**